src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -206,14 +206,14 @@
     <Compile Include="System\Globalization\CharUnicodeInfoData.cs" />
     <Compile Include="System\Globalization\GlobalizationMode.cs" />
     <Compile Include="System\Globalization\GlobalizationMode.Windows.cs" Condition="'$(TargetsWindows)'=='true'" />
-    <Compile Include="System\Globalization\GlobalizationMode.Unix.cs" Condition="'$(TargetsUnix)'=='true'" />
+    <Compile Include="System\Globalization\GlobalizationMode.Unix.cs" Condition="'$(TargetsUnix)'=='true' or '$(Platform)' == 'wasm'" />
```

Comment:
This should not be needed once we start treating wasm as Unix consistently.

Reply:
Removed